requirements.root.txt

```diff
@@ -1,2 +1,2 @@
-dask[complete]==2.8.1
-distributed==2.8.1
\ No newline at end of file
+dask[complete]==2.21.0
```

Comment:
maybe shouldn't be part of this PR

I should enable Dependabot for this repository but then we may have limited tests. I guess test in staging.